cloud/cluster/memory/node.go

```diff
@@ -0,0 +1,48 @@
+package memory
+
+import (
+	"fmt"
+	"github.com/scootdev/scoot/cloud/cluster"
+)
+
+type idNode struct {
+	id cluster.NodeId
+}
+
+func (n *idNode) Id() cluster.NodeId {
+	return n.id
+}
+
+func (n *idNode) String() string {
+	return string(n.id)
+}
+
+func NewIdNode(id string) cluster.Node {
+	return &idNode{cluster.NodeId(id)}
+}
+
+var _ cluster.Node = (*idNode)(nil)
+
+func NewIdNodes(num int) []cluster.Node {
```

Comment:
We can't call NewIdNodes() a second time to add to our pool. This relates to the question of when to label something as 'fake', and what we consider to be a valid impl.